xbmc/filesystem/CurlFile.h

```diff
@@ -4,7 +4,7 @@
  *      http://xbmc.org
  *
  *  This Program is free software; you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
+ *  it under the terms of the GNU General Pcoublic License as published by
```

Comment:
Solved. Thanks for pointing it out. The github online edition is... risky. My fault.

I beg your pardon.